src/components/HeaderSaveButton.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { HeaderButtons, Item } from 'react-navigation-header-buttons';
+import { HeaderButtonComponent } from './HeaderButtonComponent';
+
+interface Props {
+  onPress: () => void;
+}
+
+export const HeaderSaveButton = ({ onPress }: Props) => {
+  return (
+    <HeaderButtons HeaderButtonComponent={HeaderButtonComponent}>
```

Comment:
Should this use [Appbar.Action](https://callstack.github.io/react-native-paper/docs/components/Appbar/AppbarAction) instead?

Reply:
I thought the same thing, but it seems like `Appbar.Action` only allows icons (and not text like "Save").  I don't love the idea of using our chromicons save icon, or really any icon - I personally prefer the "Save" text here and anywhere else we need it.

Reply:
Ahh nice that makes sense.